### PR TITLE
remove generated code for the health endpoint

### PR DESCRIPTION
**Pull request by blampe**
I think we have a problem with `thrift.load` requiring a service name.

This makes sense for callers because they can pass in either the hostport or the name to use on Hyperbahn.

For servers it doesn't make sense. You already initialized your TChannel instance with your name/hostport, and you're just using `thrift.load` to register your handlers. The `Meta::health` endpoint here is a good example -- I pass in a service of `None` and it's totally fine.

@breerly @junchaowu

### CODE REVIEW

File: tests/test_health.py [hunk 1]
```diff
@@ -33,12 +33,14 @@ def test_default_health():
     server.listen()
 
     client = TChannel("health_test_client")
-    service = thrift_request_builder(
-        service='meta',
-        thrift_module=Meta,
+
+    service = thrift.load(
+        'health_test_server',
```
Comment (junchaowu) [inline tests/test_health.py]: little bit weird to set service name in the thrift.load.

**Comment by junchaowu** [comment]
others look good.

File: tests/test_health.py (hunk 1 above)
Comment (blampe) [inline tests/test_health.py]: This one makes sense because it's a downstream call and this is the name hyperbahn would get.

File: CHANGES.rst [hunk 2]
```diff
@@ -4,7 +4,8 @@ Changes by Version
 0.17.1 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Made "service" optional for ``thrift.load()``. The first argument should be a
+  path, but backwards compatibility is provided for 0.17.0.
```
Comment (breerly) [inline CHANGES.rst]: thanks :)

File: tchannel/thrift/rw.py [hunk 3]
```diff
@@ -37,7 +37,7 @@
 from .module import ThriftRequest
 
 
-def load(service, path, hostport=None, module_name=None):
+def load(path, service=None, hostport=None, module_name=None):
```
Comment (breerly) [inline tchannel/thrift/rw.py]: cool :)

File: tests/schemes/test_thrift.py [hunk 4]
```diff
@@ -134,8 +130,8 @@ def service(server, use_thriftrw_client):
 
     if use_thriftrw_client:
         return thrift.load(
-            'server',
             'tests/data/idls/ThriftTest.thrift',
+            'server',
```
Comment (breerly) [inline tests/schemes/test_thrift.py]: you should switch these to kwargs

File: tests/test_health.py (hunk 1 above)
Comment (breerly) [inline tests/test_health.py]: agreed w @blampe here

Comment (breerly) [inline tests/test_health.py]: lets use kwargs whenever calling thrift.load

Comment (breerly) [inline tests/test_health.py]: also we want the service name to be second

**Pull request merged**